Source/Csla/Serialization/Mobile/CustomSerializers/ClaimsPrincipalSerializer.cs

```diff
@@ -0,0 +1,39 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="ClaimsPrincipalFormatter.cs" company="Marimer LLC">
+//     Copyright (c) Marimer LLC. All rights reserved.
+//     Website: https://cslanet.com
+// </copyright>
+// <summary>A formatter that can serialize and deserialize a ClaimsPrincipal object</summary>
+//-----------------------------------------------------------------------
+
+using System.Security.Claims;
+
+namespace Csla.Serialization.Mobile.CustomSerializers;
+
+/// <summary>
+/// A formatter that can serialize and deserialize a ClaimsPrincipal object.
+/// </summary>
+public class ClaimsPrincipalSerializer : IMobileSerializer
+{
+  /// <inheritdoc cref="IMobileSerializer.Deserialize(SerializationInfo)"/>
```

Comment:
In this case you don't need the `cref` stuff because it's the method from the interface.

Reply:
So just the inherit tag by itself?

Reply:
Yes. ```// <inheritdoc />``` should be sufficient.